<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wheat_landWindow</class>
 <widget class="QWidget" name="Wheat_landWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>25</y>
     <width>300</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: rgb(170, 85, 0, 0.7);
border-radius : 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="upgrade_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>355</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ONE DAY</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="LEVEL_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string>LEVEL : </string>
   </property>
  </widget>
  <widget class="QLabel" name="level_number_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="area_number_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AREA_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string>AREA : </string>
   </property>
  </widget>
  <widget class="QLabel" name="CULTIVATED_AREA_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string>CULTIVATD ARE :</string>
   </property>
  </widget>
  <widget class="QLabel" name="cultivated_area_number_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_number_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DAYS_TO_WAIT_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string>DAYS TO WAIT : </string>
   </property>
  </widget>
  <widget class="QLabel" name="days_t_wait_number_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color: rgb(255, 170, 0);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="harvest_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>145</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ONE DAY</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Harvest</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius : 25px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color : rgb(255, 57, 57);
}</string>
   </property>
   <property name="text">
    <string>Wheat Land</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
